js/views/transactionModalVw.js

```diff
@@ -101,7 +105,11 @@ module.exports = baseVw.extend({
       success: function (model, response, options) {
         self.model.set('displayJSON', JSON.stringify(model.toJSON(), null, 2));
         //TODO set 'payout' here if the user has a payout from a dispute
-        self.model.updateAttributes();
+        if(!response.invalidData){
+          self.model.updateAttributes();
```

Comment:
So this type of functionality (calling updateAttributes), ideally, should be centralized in the model. When it's not, the model isn't self-sufficient, it needs a view to do some coordination. The problem with that is the Model in and of itself is not totally reusable. Any other view would need to duplicate this coordination.

But, it's fine for now. I understand we need to revamp models in general and once at least some of that is done we would have a blue print for stuff like this.